POP.nix

```diff
@@ -351,7 +365,15 @@
   # namePop :: String (Pop A B) -> Pop A B
   namePop = name: p: p // {__meta__ = (getMeta p) // {inherit name;};};
 
-  # Turn a pop into a normal attrset by erasing its `__meta__` information.
+  # Turn a pop into a normal attrset by erasing its `__meta__` and other invisible attributes.
   # unpop :: Pop A B -> A
-  unpop = p: builtins.removeAttrs p ["__meta__"];
+  unpop = p:
+    builtins.removeAttrs p (
```

Comment:
The `removeAttrs` becomes redundant if you already have `"__meta__"` in your bottom visibility above.

Reply:
... assuming you either include the `bottomProto` in the list, or use its visibility as the bottom of your `foldr`.

Reply:
If we have the function `unpop`, do we still want the field `__unpop__` by default?

Reply:
Yeah the bottomProto is in the list, so __meta__ will be removed during an unpop. I still think `__unpop__` is useful, because it lets pops customize their unpop function. And for code that exports a pop, a consumer doesn't have to import the pop library to unpop it, the unpopping logic is embedded in it.